### PR TITLE
env create: add a create environment command

### PR DESCRIPTION
**Pull request by gibiw**
env create: add a create environment command
--
Implemented a create environment command. Add unit tests

### CODE REVIEW

**Review by NickVolynkin — APPROVED** [review]

File: cmd/testops/env/create/create.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		description string
+		slug        string
+		host        string
+		output      string
+	)
+
+	cmd := &cobra.Command{
+		Use:     "create",
+		Short:   "Create a new environment",
+		Example: "qli testops env create --title 'New environment' --description 'This is a environment' --slug local --host app.server.com --project 'PRJ' --token 'TOKEN'",
```
Comment (NickVolynkin) [inline cmd/testops/env/create/create.go:36]: Title and slug are required, let's put them first.

```suggestion
		Example: "qli testops env create --slug stage --title 'Stage environment' --description 'Testing code before production'  --host app.server.com --project 'PRJ' --token 'TOKEN'",
```

File: cmd/testops/env/create/create.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+		},
+	}
+
+	cmd.Flags().StringVar(&title, titleFlag, "", "title of the environment")
+	err := cmd.MarkFlagRequired(titleFlag)
+	if err != nil {
+		fmt.Println(err)
+	}
+	cmd.Flags().StringVarP(&description, descriptionFlag, "d", "", "description of the environment")
+	cmd.Flags().StringVarP(&slug, slugFlag, "s", "", "slug of the environment")
```
Comment (NickVolynkin) [inline cmd/testops/env/create/create.go:74]:
```suggestion
	cmd.Flags().StringVarP(&slug, slugFlag, "s", "", "slug of the environment, (string without spaces)")
```

File: cmd/testops/env/create/create.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	err = cmd.MarkFlagRequired(slugFlag)
+	if err != nil {
+		fmt.Println(err)
+	}
+	cmd.Flags().StringVar(&host, hostFlag, "", "host of the environment")
```
Comment (NickVolynkin) [inline cmd/testops/env/create/create.go:79]:
```suggestion
	cmd.Flags().StringVar(&host, hostFlag, "", "host (URL) of the environment")
```

File: cmd/testops/env/create/create.go (hunk 1 above)
Comment (gibiw) [reply cmd/testops/env/create/create.go]: fixed

File: cmd/testops/env/create/create.go (hunk 3 above)
Comment (gibiw) [reply cmd/testops/env/create/create.go:84]: fixed

**Pull request merged**